megamek/src/megamek/client/ui/swing/ForceGeneratorViewUi.java

```diff
@@ -117,7 +118,7 @@ private void initUi() {
 
         forceTree = new JTree(new ForceTreeModel(null));
         forceTree.setCellRenderer(new UnitRenderer());
-        forceTree.setRowHeight(80);
+        forceTree.setRowHeight(0);
```

Comment:
0?

Reply:
0 set it to auto size

Reply:
Ok, let's put in a comment explaining that.